### PR TITLE
feature/COR-936-vaccination-page-table

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard26**
## Summary
* Added `VaccineCampaignsTile` component to include on the Vaccinations page
* Added generic shared table components for the vaccine campaign table
* Added new `WideVaccineCampaignTable` component for rendering the vaccine campaign table on desktop
* Added new `NarrowVaccineCampaignTable` component for rendering the vaccine campaign table on mobile

### CODE REVIEW

**Review by VWSCoronaDashboard19 — APPROVED** [review]
Pretty clean code, good work. LGTM 👍

**Review by VWSCoronaDashboard24 — CHANGES_REQUESTED** [review]
Screenshot 2022-07-19 at 12 21 47

Following the design, the number in the circled text should be bold

**Comment by VWSCoronaDashboard24** [comment]
Screenshot 2022-07-19 at 12 19 24

The order of the table headings is not correct. It should be: "Vaccinaties", "Afgelopen week", "Totaal aantal *"

**Comment by VWSCoronaDashboard24** [comment]
Screenshot 2022-07-19 at 12 20 05

The Markdown markup is missing here on both mobile and desktop

**Comment by VWSCoronaDashboard24** [comment]
Also a link to the Cijferverantwoording is missing.

File: packages/app/src/domain/vaccine/vaccine-campaigns-tile/types.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  vaccine_administered_last_week: number;
+}
+
+export type VaccineCampaignDescriptions = {
+  [key: string]: string;
+};
+
+export type VaccineCampaignHeaders = {
+  [key: string]: string;
+};
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/domain/vaccine/vaccine-campaigns-tile/types.ts:15]: Another way of writing this could be `Record<string, string>`

**Pull request merged**